describealign.py

```diff
@@ -1110,33 +1110,55 @@ def combine_gui(video_files, audio_files, config_path):
       break
   combine_window.close()
 
-def migrate_config(old_path: Path, new_path: Path) -> None:
+def migrate_config(old_paths: list[Path], new_path: Path) -> bool:
   """
   Migrate configuration from old location.
   
-  Only runs if the old_path exists but new_path does not
+  Only runs if one of old_paths exists but new_path does not
+  
+  :param old_paths: A list of previous config file locations, in chronological order oldest to newest
+  :param new_path: The new config location
+  :returns: True if the config was migrated, else False
   """
-  if new_path.exists() or not old_path.exists():
-    return
+  if new_path.exists():
```

Comment:
What about replacing this with a filter so the exception print at line 1161 isn't printing non-existent paths, for example:
```
# filter out non-existent paths
old_paths = [old_path for old_path in old_paths if old_path.exists()]
# get newest path, if it exists
old_path = None if len(old_paths) == 0 else old_paths[-1]
if new_path.exists() or old_path == None:
...
# old code here
...
return True
```
I think that should also do everything you wanted here while otherwise using the old code, adding a minimal number of new lines, and maybe being a bit more readable (at least to me)

Reply:
Fixed in 732a40d